### DISCUSSION TITLE
Multi user POS

### DISCUSSION THREAD

**Discussion by laurelkft** [nav-gov-hu/eRECEIPT#8] [Ötletek]
Nem a mi ötletünk volt a szakmai fórumon, de határozottan támogatni tudjuk azt az elképzelést, hogy multi-user pénztárgép lehessen, egy, vagy több pénzátvételi hellyel.
Ha támogatható az ötlet, mindenképpen érdekes lehetne számunkra.

**Comment by AttilaPletyak** [comment]
+1 szavazat tőlem a multi-user POS koncepcióra :) 

Itt a fő eldöntendő kérdés, hogy ilyen multi-user POS esetén lehet-e multi-kasszafiók is, azaz például egy étteremben, 3 pincér dolgozik 1 kasszára, de mindenkinél saját készpénz állomány van, azaz nem a kasszához rendelt fiókban van a készpénz. Így ha kimegy az ellenőrző hatóság, és kér a kasszán egy rovancsot, hogyan fogják tudni összevetni a kassza által regisztrált fióktartalmat a kasszánál (esetlegesen) meglévő fiókkal és a pincéreknél külön-külön meglévő készpénzzel? A szabályozásban erre ki kell térni, mind az ilyen kasszát használó kereskedő, mind az őt ellenőrző személy érdekében, hogy ne legyenek félreértések.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
